### ISSUE TITLE
Dropping jerrycan can hurt the player dropping it

### ISSUE THREAD

**Issue by captainblaffer** [KoffeinFlummi/AGM#2143]
Dropping jerrycan can hurt the player dropping it

I've had it occur to me once(for the one time I used jerrycan). Needed to morphine myself afterwards IIRC.

Did not try to reproduce later.

**Comment by jonpas** [comment]
Happened to me as well, a couple of times. Maybe a work-around would be to disable simulation when releasing it?

**Comment by commy2** [comment]
Thats Arma collision physics... Doesn't seem to happen when I release it when standing as opposed to walking.
The damage is minor too. I'd say this is very low prio.

**Comment by jonpas** [comment]
Indeed, if only happens with slightly weird movement, very minor damage.

**Comment by BaerMitUmlaut** [comment]
Damage during dropping is already disabled. However I think the damage is reenabled too quickly so I would try `waitUntil{velocity _target == [0,0,0]};` maybe?

**Comment by commy2** [comment]
That would mean that you are invulnerable when droping a jerrycan from a building. I don't like that idea.

**Comment by BaerMitUmlaut** [comment]
True. A time limit maybe? Or just a sleep?

**Comment by jonpas** [comment]
None of it sound like a good thing to implement. I would say tag it as BIS' fault (enableSimulation faulties?) and leave it as it is.

**Comment by commy2** [comment]
To much usage of `enableSimulation` is a bad idea in MP, because objects with disabled simulation no longer get synched over the net. This leads to no more position updates untill the simulation is re-enabled.
